lightning/src/ln/channelmanager.rs

```diff
@@ -4287,8 +4287,13 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	///
 	/// The `temporary_channel_id` parameter indicates which inbound channel should be accepted.
 	///
-	/// [`Event::OpenChannelRequest`]: crate::util::events::Event::OpenChannelRequest
-	pub fn accept_inbound_channel(&self, temporary_channel_id: &[u8; 32]) -> Result<(), APIError> {
+	/// For inbound channels, the `user_channel_id` parameter will be provided back in
+	/// [`Event::ChannelClosed::user_channel_id`] to allow tracking of which events correspond
+	/// with which `accept_inbound_channel` call.
+	/// 
```

Comment:
nit: here and in events.rs you have EOL whitespace. `git show` locally should highlight them for you so its easy to remove them.